src/main/scala/uk/gov/nationalarchives/dp/client/Entities.scala

```diff
@@ -1,27 +1,37 @@
 package uk.gov.nationalarchives.dp.client
 
 import cats.MonadError
+import cats.implicits.catsSyntaxOptionId
 
 import java.util.UUID
 
 object Entities {
-  case class Entity(entityType: String, ref: UUID, title: Option[String], deleted: Boolean, path: String)
+  case class Entity(
+      entityType: Option[String],
+      ref: UUID,
+      title: Option[String],
+      deleted: Boolean,
+      path: Option[String]
+  )
 
   def fromType[F[_]](entityType: String, ref: UUID, title: Option[String], deleted: Boolean)(implicit
       me: MonadError[F, Throwable]
   ): F[Entity] = entityType match {
     case "IO" =>
       me.pure {
-        Entity("IO", ref, title, deleted, "information-objects")
+        Entity("IO".some, ref, title, deleted, "information-objects".some)
       }
     case "CO" =>
       me.pure {
-        Entity("CO", ref, title, deleted, "content-objects")
+        Entity("CO".some, ref, title, deleted, "content-objects".some)
       }
     case "SO" =>
       me.pure {
-        Entity("SO", ref, title, deleted, "structural-objects")
+        Entity("SO".some, ref, title, deleted, "structural-objects".some)
+      }
+    case _ =>
```

Comment:
1. How come we're not raising an exception anymore? 
2. What if someone makes a typo?

Reply:
1. Because I don't trust the Preservica API to behave and not send us something else unexpected so I'm making this as permissive as possible and we'll see if it causese problems further down the line.
2. As above, I don't imagine this will be created by any calling clients much. It's mostly something to be returned and passed straight back in. Also, the type doesn't really matter, it's the `path` that matters because that's used to construct the API url so so long as you get that right, you're good. And if you don't, you'll get a 404.